js/change-list.js

```diff
@@ -135,10 +135,9 @@ const OP_TABLE = [
     const a = mem32[i++];
     const b = mem32[i++];
     const el = top(changeList.stack);
-    const listener = new Listener(a, b, eventType, changeList.eventsTrampoline, el);
-    changeList.listeners.add(listener);
-    el.addEventListener(eventType, listener.callback);
-    el[`dodrio-${eventType}`] = listener;
+    el.addEventListener(eventType, changeList.eventHandler);
+    el["dodrio-a-" + eventType] = a;
```

Comment:
Unless interpolation strings have some overhead that `+` doesn't, I'd stylistically prefer

```suggestion
    el[`dodrio-a-${eventType}`] = a;
```